README.md

```diff
@@ -85,6 +85,15 @@ npm install --save-dev babel-preset-es2015 babel-plugin-external-helpers
 
 The `es2015` preset includes the [transform-es2015-modules-commonjs](http://babeljs.io/docs/plugins/transform-es2015-modules-commonjs/) plugin, which converts ES6 modules to CommonJS – preventing Rollup from working. Since Babel 6.3 it's possible to deactivate module transformation with `"modules": false`. So there is no need to use the old workaround with `babel-preset-es2015-rollup`, that work for Babel <6.13. Rollup will throw an error if this is incorrectly configured.
 
+However, setting `modules: false` in your `.babelrc` may conflict if you are using `babel-register`. To work around this, specify `babelrc: false` in your gulp config. This allows Rollup to bypass your `.babelrc` file. In order to use the `es2015` preset, you will also need to specify the preset `es2015-rollup` :
+```js
+plugins: [
+  babel({
+    babelrc: false,
+    presets: ['es2015-rollup']
```

Comment:
You can just use modules false here as well though? I don't think we want to recommend es2015-rollup in the future